napari/layers/tracks/_tracks_key_bindings.py

```diff
@@ -22,7 +22,7 @@ def activate_tracks_transform_mode(layer):
 
 @register_tracks_mode_action(trans._('Pan/zoom'))
 def activate_tracks_pan_zoom_mode(layer: Tracks):
-    layer.mode = Mode.PAN_ZOOM
+    layer.mode = str(Mode.PAN_ZOOM)
```

Comment:
Hum, that seem wrong, or actually the wrong direction if we are trying to get static typing.